types/stripe-js/index.d.ts

```diff
@@ -40,6 +40,23 @@ declare module '@stripe/stripe-js' {
     /// https://stripe.com/docs/js/payment_intents
     /////////////////////////////
 
+    /**
+     * Use `stripe.confirmAcssDebitPayment` in the [Accept a payment](/docs/payments/accept-a-payment) flow with [Canadian pre-authorized debit](/docs/payments/acss-debit) payment method when the customer submits your payment form..
```

Comment:
Any links to the docs should be an absolute URL. These links will show up as clickable in the tooltips of a user's editor, but will fail to resolve if they don't include `https://stripe.com` in the beginning. 
```suggestion
     * Use `stripe.confirmAcssDebitPayment` in the [Accept a payment](https://stripe.com/docs/payments/accept-a-payment) flow with a [Canadian pre-authorized debit](/docs/payments/acss-debit) payment method when the customer submits your payment form.
```
There is also a slight typo here (need to insert "a" before "Canadian pre-authorized debit payment method") and an extra period.

I think that perhaps this could just be a single link to https://stripe.com/docs/payments/acss-debit/accept-a-payment, rather than a link to the two separate guides. Wdyt? The default "Accept a payment" guide doesn't included any information about ACSS debits at all, so it seems more confusing to link to than helpful.

Reply:
I like this idea. Rephrased to be a single link